plugins/fe_events_direct/README.md

```diff
@@ -0,0 +1,29 @@
+# Local FE Events Direct Webservice Plugin 
+
+This plugin exposes an webservice enpoint (AJAX capable) that receives events and forwards them on to an EventsAPI Server defined by environment variable EVENTSAPI_SERVER
+
+This exists in opposition to `./plugins/fe_events_moodle` which is similar but logs events through the Moodle Events API
+
+## Testing with Raise-MoodleCLI
+
+The simplest way to test this plugin is to open up the raise-moodlecli tool that can be found at https://github.com/openstax/raise-moodlecli/tree/k12-173/spike-code. This CLI contains a command - 'log-event-manually direct' which will exercise the AJAX endpoint with a phoney event. 
```

Comment:
Ah -- this link you have in the `README.md` is going to break after we merge that PR. You can just link to the root of the repo instead.